### PR TITLE
Updated ballot results and confirmed TCK download specifics

### PR DESCRIPTION
**Pull request by edbratt**
- [x] The specification committee mentor adds this final checklist to the main PR.
 - [x] The specification committee member adds the `approved` label to the PRs, and sends out the Ballot Summary per this [template](#ballot-summary-email-template) to the [public Jakarta EE Specification Committee email list](jakarta.ee-spec@eclipse.org)
  - [x] The specification committee mentor merges the specification (and apidocs) PRs, ensuring the "date:" field in the _index.md file has an appropriate value to allow publishing.
 - [x] The specification committee mentor calculates the staged EFTL TCK signature and promotes it to the committee download area
  using the https://ci.eclipse.org/jakartaee-spec-committee/job/promote-release/ job.
 - [x] The specification project member who created the api staging release promotes the specification api jars to maven central. An example release job script can be found here https://wiki.eclipse.org/MavenReleaseScript.
 - [x] The specification committee mentor updates the specification page with the ballot results.
This list goes on the committed spec index page.
 - [x] The specification project team should go through the merged spec website page to verify all the links are valid.
 - [x] The specification project team should approve the compatibility request.
 - [x] The compatible implementation project/vendor MUST send an email to tck@eclipse.org for approval of the compatible implementation for trademark usage.
 - [x] The specification project team should merge any final release branch as appropriate for the branch management for the project.

Signed-off-by: Ed Bratt <ed.bratt@oracle.com>

### CODE REVIEW

**Comment by edbratt** [comment]
@starksm64, I believe the remainder of the check-list items are for the API committer team and the compatible product team. After providing feedback on this PR, please review the check-list and thank you for finalizing those remaining details.

https://github.com/jakartaee/specifications/pull/221

File: dependency-injection/2.0/_index.md [hunk 1]
```diff
@@ -38,19 +38,20 @@ Please reference that ballot for the official results.
...
+| Andrew Pielage, Matt Gill                      | Payara              |  +1  |
+| Scott Stark, Mark Little                       | Red Hat             |  +1  |
+| David Blevins, Jean-Louis Monteiro             | Tomitribe           |  +1  |
+| Ivar Grimstad                                  | EE4J PMC            |  +1  |
+| Marcelo Ancelmo, Martijn Verburg               | Participant Members |  +1  |
+| Werner Keil                                    | Committer Members   |  +1  |
+| Scott (Congquan) Wang                          | Enterprise Members  |  +1  |
+|                                                | Total               |  10  |
 
 The ballot was run in the [jakarta.ee-spec mailing list]()
```
Comment (kwsutter) [inline dependency-injection/2.0/_index.md:57]: You should insert a link to the ballot reference from the ee-spec mailing list archive.

Comment (edbratt) [reply dependency-injection/2.0/_index.md]: Done! Thank you for the review.

**Comment by edbratt** [comment]
Thank you @starksm64 !

**Pull request merged**